backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisSession.java

```diff
@@ -15,10 +19,18 @@ public class MolgenisSession {
   private Database database;
   private GraphQL graphqlForDatabase;
   private Map<String, GraphQL> graphqlPerSchema = new LinkedHashMap<>();
+  private static final Cache<String, GraphQL> anonymousGqlObjectCache =
+      Caffeine.newBuilder()
+          .maximumSize(100)
+          .expireAfterAccess(1, TimeUnit.MINUTES)
```

Comment:
So if it is accessed it will stay get an extra minute? before invalidating? Isn't that a bit short